app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+class ApplicationPolicy
```

Comment:
I like it - it seems like a very straightforward way to define authorization roles. It'll be interesting to work with it further.